### PR TITLE
Catalog tabs

### PR DESCRIPTION
**Pull request by sufeiiz**
I renamed a log of fragments, because naming is always hard. Please help me rename `ProgramCatalogCatalogSeries` because I really don't like it.

To test out this build, run on your emulator and click on "Find Program" on the bottom nav bar

### CODE REVIEW

File: app/src/main/java/com/superfit/android/catalog/CatalogSectionTabAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.superfit.android.catalog
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentStatePagerAdapter
+
+class CatalogSectionTabAdapter(
```
Comment (leojkwan) [inline app/src/main/java/com/superfit/android/catalog/CatalogSectionTabAdapter.kt]: Hi @sufeiiz just curious— what is the role of an "Adapter" in this context?

File: app/src/main/java/com/superfit/android/data/ProgramCatalog.kt [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 
-import android.arch.persistence.room.ColumnInfo
-import android.arch.persistence.room.Entity
-import android.arch.persistence.room.PrimaryKey
+import java.io.Serializable
 
-@Entity(tableName = "programCatalog")
 data class ProgramCatalog(
-        @PrimaryKey val id: Int,
+        val id: String,
```
Comment (leojkwan) [inline app/src/main/java/com/superfit/android/data/ProgramCatalog.kt]: keep primary key no?

**Comment by leojkwan** [comment]
https://superfit.nyc3.cdn.digitaloceanspaces.com/misc/screencast%202018-12-18%2015-33-49.mp4

Not sure if I built the project correct, but I may have crashed the app?

File: app/src/main/java/com/superfit/android/data/ProgramCatalog.kt (hunk 2 above)
Comment (sufeiiz) [reply app/src/main/java/com/superfit/android/data/ProgramCatalog.kt]: I am saving only saving catalog section as a whole in the database right now, which is why in the dao you can only search by catalog section. We can later on add a programCatalogDao then add primary keys for that.

**Comment by sufeiiz** [comment]
@leojkwan added some changes, see if its still crashing for you

**Pull request merged**